### DISCUSSION TITLE
Transfer GitHub Education to another GitHub Account

### DISCUSSION THREAD

**Discussion by userari** [community/community#110988] [GitHub Education]
### Hi everyone,

I'm wondering if it's possible to transfer the benefits of a GitHub student account to an account that doesn't have student benefits. 
All my work is on the account without the benefits, and I plan to delete the account with the benefits because I don't use it. I really don't want to merge unless there's no other choice.
I've also seen that it's advised to keep only the account with the benefits, but is it possible to do the opposite and transfer it by deleting the account with the benefits? So I can have it on the other account 😄 

Thank you very much, and sorry if it's confusing.

**Comment by SebastienXia** [comment]
## Hi, @userari 

For your problem I haven't found anything better than [the documentation on account transfer](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts)

### Open ticket to a support 
You can also open a ticket for this [here](https://support.github.com/)
